my-app/src/components/content/content.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import './content.css';
+
+const Content = () => {
+
+    return (
+        <div className="content">
+            <div className="container">
+                <div className="wrapper-product">
+                    <div className="product-item">
```

Comment:
make file constants.js. Create there array with your data. Use .map method to show your items